### DISCUSSION TITLE
Trigger Remote Tests on LHCI server?

### DISCUSSION THREAD

**Discussion by francis826** [GoogleChrome/lighthouse-ci#703] [Q&A] [answered]
Hello,

I have an LHCI server running in docker on a remote VM, can I use my local machine to trigger that docker LHCI server (compared to running lhci collect locally) to run tests against a specific URL, collect, and display results? I've scraped the documentation but from what I can tell it only seems possible to run the tests locally and upload them to the server. Am I missing something obvious?

Thanks!

**Comment by patrickhulce** [comment, marked as answer]
> Am I missing something obvious?

Maybe :)

The purpose of the server is to *store* your results from ephemeral continuous integration VMs that are doing the actual testing, not  run tests for you, so no, there is no concept of "triggering" a test on the server from a local machine as the server is not intended to run Lighthouse on a URL.

There is however, the concept of configuring the server to periodically request PageSpeed Insights API to do this for you in the option [`psiCollectCron`](https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md#psicollectcron).

**Reply by francis826** [reply]
Thank you Patrick for confirming my suspicion!
